lib/services/project-changes-service.ts

```diff
@@ -227,11 +232,8 @@ export class ProjectChangesService implements IProjectChangesService {
 
 			let fileStats = this.$fs.getFsStats(filePath);
 
-			let changed = fileStats.mtime.getTime() >= this._outputProjectMtime || fileStats.ctime.getTime() >= this._outputProjectCTime;
-			if (!changed) {
-				let lFileStats = this.$fs.getLsStats(filePath);
```

Comment:
it's unnecessary to do `this.$fs.getLsStats(filePath)` twice and the logic is kept the same in the refactored version